packages/plugin-flex/src/sub-commands/create-configuration.ts

```diff
@@ -54,16 +23,53 @@ delete baseFlags.json;
  * Creates a Configuration
  */
 export default abstract class CreateConfiguration extends FlexPlugin {
+  static topicName = 'flex:plugins:create-configuration';
+
+  static description = createDescription(CreateConfiguration.topic.description, true);
+
+  static nameFlag = {
+    description: CreateConfiguration.topic.flags.name,
+    default: `Autogenerated Release ${Date.now()}`,
+    required: true,
+    max: 100,
+  };
+
+  static enablePluginFlag: Partial<flags.IOptionFlag<string[]>> & Multiple = {
```

Comment:
what does `& Multiple` mean 😯

Reply:
It's the union operator - make `enablePluginFlag` a combination of `Partial<flags.IOptionFlag<string[]>>` and `Multiple`

Reply:
oh i totally misread and thought `Multiple` was a special typescript thing 😂